requirements/requirements_container.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements_base.txt
+git+https://github.com/mpi4py/mpi4py.git@aac3d8f2a56f3d74a75ad32ac0554d63e7ef90ab
```

Comment:
I think this is an interesting question. Does it have to match?

Reply:
Not necessarily, in some cases might not be able to, but the less version divergence we have in different environments, the less confusing it is to support multiple environments.

Reply:
I don't think this is too much of an issue. Remember that this is temporary because we are using a feature in the non-released `mpi4py`. E.g we don't need this commit as much as "this commit at least'. Locking it in a commit that works is good practice but this should go away if they ever decide to release 3.1.0